doc/runtimes/nodejs.md

```diff
@@ -55,6 +55,9 @@ If a `yarn.lock` file is found in your repo root or if you specify "yarn" in
 the `engines` field of package.json, the latest or specified version of yarn
 will be used instead of npm.
 
+Note that **installing packages globally is unsupported**, whether requested directly
```

Comment:
sugg: add an example to make it clearer, e.g. `npm install -g`, recommending that all packages are included package.json.